sql/snapshot.sql

```diff
@@ -0,0 +1,100 @@
+-- public.global_snapshot definition
+
+-- Drop table
+
+-- DROP TABLE global_snapshot;
+
+CREATE TABLE global_snapshot (
+	ordinal int8 NOT NULL,
+	hash text NOT NULL,
+	height int8 NOT NULL,
+	subheight int8 NOT NULL,
+	last_snapshot_hash text NULL,
+	created_at timestamp NULL,
+	updated_at timestamp NULL,
+	CONSTRAINT global_snapshot_pkey PRIMARY KEY (ordinal)
+);
+CREATE INDEX global_snapshot_hash_idx ON public.global_snapshot USING btree (hash);
+
+
+-- public.dag_balance_change definition
+
+-- Drop table
+
+-- DROP TABLE dag_balance_change;
+
+CREATE TABLE dag_balance_change (
```

Comment:
Notes: 
- `ordinal` - since we have global ordinals and metagraph/currency ordinals I would suggest making each explicit every time we use it. This would become `global_ordinal`.
- `address` can be VARCHAR
- This will be queried by `address` and then `ordinal` frequently so we probably need an index on `(address, ordinal)` w/address first. I think we can just make that the order of the PK and that would work.
- Do we want to have an `addresses` table that this FKs to? We’ll have several tables that reference `address`.
- Should we add `created_at` and `updated_at` to all tables? Or what’s the thought behind which tables get those columns and which don’t?